README.md

```diff
@@ -22,12 +22,12 @@ according to the following naming scheme:
 Additional required information, like if the configuration includes
 biogeochemistry, is appended to the name.
 
-Currently the following configurations are available:
+Currently the following development configurations are available:
```

Comment:
here and below - are we not wanting to publicise the non-`dev-` branches in the README at this stage?